package.json

```diff
@@ -456,9 +456,45 @@
             "end"
           ]
         },
+        "mssql.format.alignColumnDefinitionsInColumns": {
+          "type": "boolean",
+          "description": "Should column definitions be aligned?",
```

Comment:
I'm trying to match other formatter settings. None of these use the [Optional] tag and it's more concise. I'd prefer to keep it this way since, by definition, formatter settings are optional.

![image](https://cloud.githubusercontent.com/assets/10819925/22997461/4da33e86-f387-11e6-94fe-06cdfb60e894.png)